.storybook/foundations/corner-rounding/corner-rounding.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta, Canvas, Story, Title } from '@storybook/blocks';
+import * as CornerRoundingStories from './corner-rounding.stories.js';
+import ComponentSizeTokensSmall from './component-size-tokens-small.mdx';
+import ComponentSizeTokensMedium from './component-size-tokens-medium.mdx';
+import * as CheckboxCornerRoundingStories from './checkbox-corner-rounding.stories.js';
+import * as ActionButtonCornerRoundingStories from './action-button-corner-rounding.stories.js';
+
+<Meta of={CornerRoundingStories} />
+
+<Title />
+
+## Alias tokens
+
+<Story of={CornerRoundingStories.CornerRounding} />
+
+## Component size tokens
+
+In addition to the alias tokens listed above, Spectrum 2 also provides component size alias tokens that align with global size tokens:
+
+### Small component size tokens
+
+<ComponentSizeTokensSmall />
+
+### Medium component size tokens
+
+<ComponentSizeTokensMedium />
+
+These component size tokens are used to scale corner rounding in different-sized tokens. For instance, the corner-rounding for a checkbox will be `4px` for a medium-sized checkbox, `3px` for small, `5px` for large, and `6px` for an extra-large checkbox, corresponding to the small component size tokens table above.
+
+### Checkbox corner rounding
+
+Checkbox will use small-sized component size tokens to scale corner rounding.
+
+Note that the story below has not yet been updated to show corner rounding that scales with component size.
```

Comment:
[Design wants to show the scaling for corner-rounding on different components](https://adobedesign.slack.com/archives/GQ09RCBA5/p1708116176602529) by showing components in different sizes with differently-sized corner-rounding, but our components don't have this baked into them yet. I've written down as a to do for myself to make sure this part of the documentation gets updated once those components are migrated and displaying as design intended.